argcomplete/_check_console_script.py

```diff
@@ -26,12 +29,11 @@ def main():
     # Find the module and function names that correspond to this
     # assuming it is actually a console script.
     name = os.path.basename(script_path)
-    entry_points = list(iter_entry_points('console_scripts', name))
+    entry_points = [ep for ep in importlib_entry_points()["console_scripts"] if ep.name == name]
     if not entry_points:
         raise ArgcompleteMarkerNotFound('no entry point found matching script')
-    [entry_point] = entry_points
-    module_name = entry_point.module_name
-    function_name = entry_point.attrs[0]
+    entry_point = entry_points[0]
```

Comment:
I originally wrote this to fail if there are two entry points for the same name found, since I don't know what actually happens in that case. I suspect one will overwrite the other, and then we don't really have a way of knowing which one to check for the marker. Perhaps we should throw `ArgcompleteMarkerNotFound` if there's more than one?

Reply:
On my computer, I get multiple entry points for packages that I have installed multiple times. I'm not sure what causes that, but it doesn't seem to me that we need to treat it as an error.

Reply:
There's nothing that stops multiple packages specifying the same `entry_point` name, in which case you could be checking for the marker in one script but executing another.